src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -292,6 +295,15 @@ const ProfilePageContent = ({
             {/* <Section title="Frequent Collaborators">TODO.</Section> */}
           </Box>
         )}
+        {isFeatureEnabled('activity') && (
+          <Box>
+            <RecentActivityTitle />
+            <ActivityList
+              queryKey={['profile-activities', profile.id]}
+              where={{ target_profile_id: { _eq: profile.id } }}
```

Comment:
lets add contributions here too...
THe UI needs to show the circle tho
```suggestion
              where={{
                _or: [
                  { target_profile_id: { _eq: profile.id } },
                  { actor_profile_id: { _eq: profile.id } },
                ],
              }}

```

Reply:
Strange, in my memory I did this